### ISSUE TITLE
Error with pyunit on osx lion

### ISSUE THREAD

**Issue by jcrudy** [pymc-devs/pymc3#142]
I'm happy to provide more details on my platform if helpful.  Just ask for what you need.  I'm using Python 2.7.

The stacktrace I get, after using MCMC.sample within a pyunit test, is pasted at the bottom of this message.  The problem seems to be related to the chr(27) printed out by the ProgressBar class.  I fixed it for myself by changing line 29 of progressbar.py from:

print self, chr(27) + '[A'

to:

print self

I assume that chr(27) is in there for a good reason, but it definitely causes an error on my system.

Best,

Jason

```
Traceback (most recent call last):

  File "/Applications/eclipse/plugins/org.python.pydev_2.7.0.2012080220/pysrc/pydev_runfiles_xml_rpc.py", line 131, in run

    self.server.notifyCommands(commands)

  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xmlrpclib.py", line 1224, in __call__

    return self.__send(self.__name, args)

  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xmlrpclib.py", line 1575, in __request

    verbose=self.__verbose

  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xmlrpclib.py", line 1264, in request

    return self.single_request(host, handler, request_body, verbose)

  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xmlrpclib.py", line 1297, in single_request

    return self.parse_response(response)

  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xmlrpclib.py", line 1473, in parse_response

    return u.close()

  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xmlrpclib.py", line 793, in close

    raise Fault(**self._stack[0])

Fault: <Fault 0: 'Failed to parse XML-RPC request: An invalid XML character (Unicode: 0x1b) was found in the element content of the document.'>
```

**Comment by jcrudy** [comment]
I've now had the same thing happen in Ubuntu as well and was able to fix it the same way.

**Comment by fonnesbeck** [comment]
Fixed in f2c862221c3372125cf354cea1a7c103ace78a4a. Please test.
